packages/phone-number-privacy/common/src/utils/authentication.ts

```diff
@@ -84,27 +91,33 @@ export async function isVerified(
   contractKit: ContractKit,
   logger: Logger
 ): Promise<boolean> {
-  return retryAsyncWithBackOff(
-    async () => {
-      const attestationsWrapper: AttestationsWrapper = await contractKit.contracts.getAttestations()
-      const {
-        isVerified: _isVerified,
-        completed,
-        numAttestationsRemaining,
-        total,
-      } = await attestationsWrapper.getVerifiedStatus(hashedPhoneNumber, account)
+  try {
+    return retryAsyncWithBackOffAndTimeout(
+      async () => {
+        const attestationsWrapper: AttestationsWrapper = await contractKit.contracts.getAttestations()
```

Comment:
do these calls to get contract wrappers perform a forno request? If so we might want to put them in their own retryAsyncWithBackOffAndTimeout() since if only the second forno request fails we'll be retrying both

Reply:
Good question. It looks like it does make a call to forno once upon initialization but the result is cached from that point onward so I don't think it's anything to worry about.